autoload/neomake.vim

```diff
@@ -577,6 +577,7 @@ function! neomake#MakeHandler(job_id, data, event_type) abort
 
         " Show the current line's error
         call neomake#CursorMoved()
```

Comment:
I think this line is not really necessary.

Only calling `neomake#EchoCurrentError` (added below) seemed to be enough (see my diff).